draft-ietf-tls-dtls13.md

```diff
@@ -750,6 +765,58 @@ because an attacker will have difficulty forging a datagram that will
 not be rejected by the transport layer.
 
 
+### AEAD Limits
+
+Section 5.5 of TLS {{!TLS13}} defines limits on the number of records that can
+be protected using the same keys. These limits are specific to an AEAD
+algorithm, and apply equally to DTLS. Implementations SHOULD NOT protect more
+records than allowed by the limit specified for the negotiated AEAD.
```

Comment:
One might mention here that the recommended action is to issue a `KeyUpdate` to update the keys (in particular, it's not necessary to terminate the connection).

Reply:
Willdo.